### ISSUE TITLE
Bug: `<animatedSprite>` doesn't support `animationSpeed`

### ISSUE THREAD

**Issue by trezy** [pixijs/pixi-react#552]
### Current Behavior

When using `<animatedSprite animationSpeed={1}>`, Typescript reports that `animationSpeed` doesn't exist.

### Expected Behavior

`animationSpeed` is supported on `<animatedSprite>`.

### Steps to Reproduce

1. Use `<animatedSprite>` in an app with Typescript and `@pixi/react`
2. Add `animationSpeed` to the component
3. Observe Typescript error

### Environment

- **`@pixi/react` version**: 8.0.0-beta.14
- **`pixi.js` version**: 8.2.6

**Comment by thejustinwalsh** [comment]
AnimatedSprite's constructor options are defined as `Texture[] | FrameObject[]`, and the parent types constructor options do not appear to be being picked up by the current method of extracting props from constructor options.

Do we also need to ensure we add the props from the class's setters / writable fields?

**Comment by thejustinwalsh** [comment]
[Pixi v8.3.0](https://github.com/pixijs/pixijs/releases/tag/v8.3.0) now has AnimatedSpriteOptions as a consturctor: https://github.com/pixijs/pixijs/commit/24d857ef7cfed25d49c83806b7186a1f04aa2c06

Bumping the min version in the deps, and adding `AnimatedSpriteOptions` to constructor overrides is the move.

LMK when you want to version bump, and I'll make the PR for the AnimatedSprite ConstructorOptions.
Typesript only allowing you to extract the ConstructorParameters of the last constructor defined is less than ideal.

**Comment by trezy** [comment]
Digging into this issue, it looks like `AnimatedSpriteOptions` only includes `textures` and `autoUpdate`, not the many other configurable options that one would expect it to support (`animationSpeed`, `loop`, `updateAnchor`, `onComplete`, `onFrameChange`, `onLoop`). I'm digging into the reason behind that and possibly getting it merged into core before we can fix this.

**Comment by trezy** [comment]
Once this PR is merged, we'll be able to bump the minimum version of core. 😉
https://github.com/pixijs/pixijs/pull/11241

**Comment by trezy** [comment]
This is solved by upgrading to Pixi.js v8.8.0+
